src/openforms/utils/urls.py

```diff
@@ -13,6 +14,9 @@
 
 from openforms.typing import AnyRequest
 
+if TYPE_CHECKING:
+    from django.conf.urls import _IncludedURLConf
```

Comment:
Is it good that we are using Django's private API here?

Reply:
this is not actually private API, but types defined by `django-stubs`! this import does not exist at runtime, and that's why it's guarded by the `if TYPE_CHECKING:` in combination with `from __future__ import annotations`